docs/getting-started/run-applications.md

```diff
@@ -35,6 +35,16 @@ on-demand by Crossplane.
 
 ## Infrastructure Operator
 
+### Install OAM Controllers
+
+Use Helm 3 to update our Crossplane installation to install OAM controllers:
+
+```console
+helm upgrade crossplane --namespace crossplane-system crossplane-alpha/crossplane --set oam.enabled=true
```

Comment:
I think we need a tab for `Helm 3 (master)` since if you installed Crossplane using `master` then the upgrade should be using the same chart version.

Reply:
Good call :+1:

Reply:
Should we just remove this section and advise folks to use the `--set oam.enabled=true` in the mainline install docs?